### PR TITLE
AhjoProposalsSrv implemented

### PR DESCRIPTION
**Pull request by evidentti**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -93,4 +96,5 @@
     <!-- APP CONTENT -->
     <ui-view name="appView"></ui-view>
 </body>
-</html>
+
+</html>
```
Comment (tkenakka) [inline app/index.html:100]: No newline at the end of file, that's usually required

File: app/index.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
```
Comment (tkenakka) [inline app/index.html:4]: Please fix the code indentations :)

File: app/scripts/services/ahjo.proposals.srv.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        var self = this;
+
+        self.getMeeting = function (guid) {
+            $log.debug("AhjoProposalsSrv: getMeeting " + guid);
+            var deferred = $q.defer();
+            $timeout(function () {
+                deferred.notify({});
+                $http({
+                    method: 'GET',
+                    cache: true,
```
Comment (tkenakka) [inline app/scripts/services/ahjo.proposals.srv.js]: Please disable the cache by default. Always fetch the latest, right?

File: app/scripts/services/ahjo.proposals.srv.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                    url: ENV.AhjoApi_Proposals.replace('{GUID}', guid)
```
Comment (tkenakka) [inline app/scripts/services/ahjo.proposals.srv.js:27]: Please remember to commit also the API URL constant `AhjoApi_Proposals` to appropriate repo.

File: app/index.html (hunk 2 above)
Comment (evidentti) [inline app/index.html:4]: This is result of Visual Code formatting tool. We decided to use it and result looks ok to me.

File: app/scripts/services/ahjo.proposals.srv.js (hunk 4 above)
Comment (evidentti) [inline app/scripts/services/ahjo.proposals.srv.js:27]: Pull request is pending on bitbucket repository.

**Comment by tkenakka** [comment]
LGTM

**Pull request merged**